packages/env/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@wordpress/env",
-	"version": "1.0.0",
+	"version": "0.0.0",
```

Comment:
Is this change intentional?

Reply:
Yes, so that the release tool can bump it for the first release.